chef/cookbooks/crowbar/files/default/crowbar.service

```diff
@@ -1,15 +1,29 @@
 [Unit]
-Description=Crowbar Daemon
+Description=Crowbar
 After=network.target syslog.target remote-fs.target chef-server.service
 
 [Service]
 Type=simple
 User=crowbar
 Group=crowbar
-PIDFile=/var/run/crowbar/crowbar.pid
+KillMode=control-group
+TimeoutSec=300
+TimeoutStopSec=10
+
+WorkingDirectory=/opt/dell/crowbar_framework
+
+Environment="CROWBAR_ENV=production"
+Environment="CROWBAR_THREADS=1"
+Environment="CROWBAR_WORKERS=5"
```

Comment:
tboerger told me he wants to have 5 workers and he didn't complained about the number of threads. But I don't know if he didn't see it.